tutorials/ports.html

```diff
@@ -250,6 +250,42 @@ <h3 id="create">Creating elements with ports</h3>
         attrs: { label: { text: 'out' }}
     }
 ]);
+</code></pre>
+
+            <h3 id="port-prop">Setting port properties</h3>
+
+            <p>
+                If you are already familiar with the <a href="/docs/jointjs#dia.Element.prototype.attr" target="_blank"><code>attr()</code></a>
```

Comment:
The counterpart of the `portProp()` function is the `prop()` function, not `attr()`.